templates/docs/process/rhpam713-authoring-ha.adoc

```diff
@@ -53,7 +53,7 @@ https://docs.okd.io/latest/architecture/core_concepts/templates.html#parameters[
 |`KIE_SERVER_HTTPS_PASSWORD` | `HTTPS_PASSWORD` | The password for the keystore and certificate for KIE Server. | mykeystorepass | False
 |`APPFORMER_JMS_BROKER_USER` | `APPFORMER_JMS_BROKER_USER` | The user name for connecting to the JMS broker. | jmsBrokerUser | True
 |`APPFORMER_JMS_BROKER_PASSWORD` | `APPFORMER_JMS_BROKER_PASSWORD` | The password to connect to the JMS broker. | -- | True
-|`DATAGRID_IMAGE` | -- | DataGrid image. | registry.redhat.io/datagrid/datagrid-8-rhel8:1.2 | True
+|`DATAGRID_IMAGE` | -- | DataGrid image. | registry.redhat.io/datagrid/datagrid-8-rhel8:1.1 | True
```

Comment:
[versioning] Is the change from 8:1.2 to 8:1.1 correct?

Reply:
it will be changed again later before the release, if needed.